### DISCUSSION TITLE
Change the color of items in Programs?

### DISCUSSION THREAD

**Discussion by stani63** [Open-Shell/Open-Shell-Menu#1801] [Q&A] [answered]
How to change the color of items in Programs? Some items have this yellow-orange color, I don't like that, I would like it in solid gray.
My settings are: Classic Style, Classic Skin.

**Comment by ge0rdi** [comment, marked as answer]
You can turn off `Highlight newly installed programs` option.

**Reply by stani63** [reply]
Thanks!
